Intersect (Core)/GameObjects/LightBase.cs

```diff
@@ -68,7 +68,11 @@ Color color
         /// <returns></returns>
         public override int GetHashCode()
         {
-            return Intensity ^ (int)Expand ^ Color?.A ?? 0 ^ Color?.R ?? 0 ^ Color?.G ?? 0 ^ Color?.B ?? 0;
+            return Intensity ^ (int)Expand ^ Color?.A
+                ?? 0 ^ Color?.R
+                ?? 0 ^ Color?.G
+                ?? 0 ^ Color?.B
+                ?? 0;
```

Comment:
This should be reverted or the `Color?.<X> ?? 0` segments need to have parentheses for readability and forced onto 1 line per parenthesized group.

That or honestly, this could change to `Intensity ^ (int)Expand ^ (Color?.GetHashCode() ?? 0)`